src/main/java/Event.java

```diff
@@ -0,0 +1,14 @@
+import java.sql.Date;
+
+public class Event {
+		Date date;
+		String time;
+		Staff staff;
+
```

Comment:
Who will attend the event ? If its just for the staff members then there should be a host and list of attendees staff members  OR  if it is related to an ongoing class then there should be a list of Students here.